nodecfg/node_config.go

```diff
@@ -802,6 +803,23 @@ func SetLogLevel(db *sql.DB, logLevel string) error {
 	return err
 }
 
+func GetNimbusTrustedBlockRoot(db *sql.DB) (string, error) {
+	var blockRoot sql.NullString
+	err := db.QueryRow("SELECT nimbus_proxy_trusted_block_root FROM node_config").Scan(&blockRoot)
+	if err != nil {
+		return "", err
+	}
```

Comment:
Is it possible for this query to return no rows and be valid? because you might get a `sql.ErrNoRows` so I wonder if we should add a 
```
	if errors.Is(err, sql.ErrNoRows) {
		return "", nil
	}
```